upstream/main build error

I think it was merged during CI fix. I am working on it
```
../nntrainer/layers/layer_node.h:120:5: error: ‘nntrainer::LayerNode::activation_type’ will be initialized after [-Werror=reorder]
  120 |     activation_type; /**< activation applied to the output of this node */
      |     ^~~~~~~~~~~~~~~
../nntrainer/layers/layer_node.h:118:8: error:   ‘bool nntrainer::LayerNode::flatten’ [-Werror=reorder]
  118 |   bool flatten; /**< flatten the output of this node */
      |        ^~~~~~~
../nntrainer/layers/layer_node.h:32:3: error:   when initialized here [-Werror=reorder]
   32 |   LayerNode(std::shared_ptr<nntrainer::Layer> l, size_t idx) :
      |   ^~~~~~~~~
../nntrainer/layers/layer_node.h:118:8: error: ‘nntrainer::LayerNode::flatten’ will be initialized after [-Werror=reorder]
  118 |   bool flatten; /**< flatten the output of this node */
      |        ^~~~~~~
../nntrainer/layers/layer_node.h:113:5: error:   ‘std::shared_ptr<nntrainer::Layer> nntrainer::LayerNode::layer’ [-Werror=reorder]
  113 |     layer;      /**< The actual object in the graph node */
      |     ^~~~~
../nntrainer/layers/layer_node.h:32:3: error:   when initialized here [-Werror=reorder]
   32 |   LayerNode(std::shared_ptr<nntrainer::Layer> l, size_t idx) :
      |   ^~~~~~~~~
```

--- a/nntrainer/layers/layer_node.h
+++ b/nntrainer/layers/layer_node.h
@@ -30,10 +30,10 @@ public:
    *
    */
   LayerNode(std::shared_ptr<nntrainer::Layer> l, size_t idx) :
-    activation_type(ActivationType::ACT_NONE),
-    flatten(false),
     layer(l),
-    index(idx) {}
+    index(idx),
+    flatten(false),
+    activation_type(ActivationType::ACT_NONE) {}
 
   /**
    * @brief     Destructor of LayerNode Class

